app/javascript/components/App.js

```diff
@@ -66,6 +66,18 @@ class App extends React.Component {
     .catch(errors => console.log("Apartment update errors:", errors))
   }
 
+  deleteApartment = (id) => {
+    fetch(`http://localhost:3000/apartments/${id}`, {
```

Comment:
Your fetch can just be `/apartments/${id}` since you are not going across origin